README.md

```diff
@@ -1 +1,22 @@
-# device-images
\ No newline at end of file
+OpenBMC power supply firmware
+
+This directory contains P11 power supply firmware images supplied by the
+vendors. Each PSU firmware should reside in a directory named as the PSU CCIN.
+The firmware image file prefix should be replaced with IBM PSU CCIN and postfix
+must remain as is. The PSU CCIN dir should contains the firmware file and
+MANIFEST file. The contents of CCIN dir can be generated in 2 ways:
+
+* - Use tool to Generate Tarball with PSU image and MANIFEST Script, the
```

Comment:
This is kind of a run-on sentence with some missing words.  Also, why is "Generate Tarball" capitalized?

Possible rewording:

Use the tool at phosphor-psu-code-mgmt/tools/generate-psu-tar to generated a tarball with the PSU image and MANIFEST file.  Then extract the tarball into the PSU CCIN directory.